tests/e2e/service_plans_test.go

```diff
@@ -46,6 +46,38 @@ var _ = Describe("Service Plans", func() {
 			})))
 		})
 	})
+	Describe("Get", func() {
+		var (
+			planGUID string
+			result   servicePlanResource
+		)
+
+		BeforeEach(func() {
+			plans := resourceList[resource]{}
+			resp, err = adminClient.R().SetResult(&plans).Get("/v3/service_plans")
+			Expect(err).NotTo(HaveOccurred())
+			Expect(resp).To(HaveRestyStatusCode(http.StatusOK))
+
+			brokerPlans := itx.FromSlice(plans.Resources).Filter(func(r resource) bool {
+				return r.Metadata.Labels[korifiv1alpha1.RelServiceBrokerGUIDLabel] == brokerGUID
+			}).Collect()
+
+			Expect(brokerPlans).NotTo(BeEmpty())
+			planGUID = brokerPlans[0].GUID
+		})
+
+		Describe("Get Service Plan", func() {
```

Comment:
This `Describe` does not alter the parent `Describe` context, therefore it makes no sense. I.e.

```
Describe("Get"....
  BeforeEach(...
  JustBeforeEach(....
  It("returns the service plan", ...
```

Reply:
at some point I will learn it >.<

Reply:
adjusted @danail-branekov